face_client/css/app.css

```
@@ -16167,4 +16179,53 @@ span.flatpickr-weekday {
+.canvas-styled:hover {
+    transform: scale(1.02);
+    box-shadow: 0px 6px 15px rgba(0, 0, 0, 0.4);
+}
+
+.d-none {
+    display: none;
+}
+
+.logo_image {
+    height: 100%;
+    weight: 100%;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)